harvester/Exporter.py

```diff
@@ -88,7 +88,6 @@ def _generate_gmeta(self, batch_size, export_filepath, temp_filepath, only_new_r
 				self.logger.debug("Writing batch %s to output file" % (gmeta_batches))
 				gingest_block = {"@datatype": "GIngest", "@version": "2016-11-09", "source_id": "ComputeCanada", "ingest_type": "GMetaList", "ingest_data": {"@datatype": "GMetaList", "@version": "2016-11-09", "gmeta":gmeta}}
 				self._write_to_file(json.dumps(gingest_block), export_filepath, temp_filepath, "gmeta", gmeta_batches)
-				del gmeta[:batch_size]
```

Comment:
OK, hang on a second ... did we lose track of the `batch_size` variable entirely? Shouldn't we be using this same behaviour with `buffer_size` now?

Reply:
Batch size isn't used, and the code was dead.  The buffer limit is being used now.  And we don't need to remove it from the gmeta dict, because it isn't in there to begin with.

Reply:
Right, but where do we drop entries from the gmeta list after a write out now? It seems like without that every subsequent write will contain all the entries from the preceding one...

Reply:
I'm confused because I was sure that we had working behaviour for both batches and buffers and I can see that the former was dead but I'm not sure where the latter is now.

Reply:
Should we just move line 63 down to 87? Unless I'm missing something?

Reply:
Good catch on purging gmeta after dump, thanks

Reply:
Yeah, I think this must have been broken-but-still-working-somehow, such that calling del gmeta[:batch_size] was actually wiping the list (due to batch size always being > the number of entries in the list).